api/sip/client.py

```diff
@@ -345,7 +345,11 @@ def __init__(
             # We're implicitly logged in.
             self.must_log_in = False
         self.login_password = login_password
-        self.dialect_config = dialect.config
+
+        if isinstance(dialect, str):
+            self.dialect_config = Dialect(dialect).config
```

Comment:
This fix here also addresses a problem with the AG Verso dialect.  Without this conversion from a string value to the enum an exception will be thrown when trying to access the non-existent property.

Reply:
This is a good catch @dbernstein.

I spent a bit of time looking into it, and this is actually a more pervasive issue then just in this integration. Instead of adding the code to deal with the dialect coming in as a string here, I'd rather deal with the up-steam cause so that we get the correct enum type passed in here.

I pushed a new PR here that should do that: https://github.com/ThePalaceProject/circulation/pull/1500